lib/tasks/limber.rake

```diff
@@ -572,8 +580,10 @@ namespace :limber do
         Limber::Helper::TemplateConstructor.new(prefix: prefix,
                                                 catalogue: catalogue,
                                                 sequencing_keys: params[:sequencing_list]).build!
-        Limber::Helper::LibraryOnlyTemplateConstructor.new(prefix: prefix, catalogue: catalogue).build!
-        Limber::Helper::LibraryAndMultiplexingTemplateConstructor.new(prefix: prefix, catalogue: catalogue).build!
+        unless params[:omit_library_templates]
+          Limber::Helper::LibraryOnlyTemplateConstructor.new(prefix: prefix, catalogue: catalogue).build!
+          Limber::Helper::LibraryAndMultiplexingTemplateConstructor.new(prefix: prefix, catalogue: catalogue).build!
+        end
```

Comment:
Did you verify that they didn't want these 'library only' and 'library and multiplexing' templates, on the call?

Reply:
Yes we verified they only wanted the 3 included submission templates, but I will check this

Reply:
It has been confirmed that the library only & library and multiplexing (Limber-Htp - pWGS and Limber-Htp - pWGS - Pool) submission templates will not be required